queries/hyprlang/highlights.scm

```diff
@@ -0,0 +1,60 @@
+(comment) @comment
+
+[
+  "source"
+  "exec"
+  "exec-once"
+] @keyword
+
+(keyword
+  (name) @keyword)
+
+(assignment
+  (name) @property)
+
+(section
+  (name) @module)
+
+(section
+  device: (device_name) @type)
+
+(variable) @variable
+
+(variable
+  "$" @punctuation.special)
+
+(number) @number
+
+(boolean) @boolean
+
+(mod) @constant
+
+[
+  "rgb"
+  "rgba"
+] @function.builtin
+
+(color
+  (legacy_hex) @number)
+
+(angle) @number
```

Comment:
Same here, if `legacy_hex` is used only here, then there's no need to be specific, and on top of that, you can combine the number queries

```scm
[
  (number)
  (legacy_hex)
  (angle)
  (hex)
] @number
```